src/pages/signIn/searchPage/components/Atoms/SearchTypeSelector/SearchTypeSelector.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+
+import styles from './SearchTypeSelector.module.css';
+//optionally future import { useTheme }? (Dark Theme) and styles below
+
+const SearchTypeSelector = () => {
+  //const { theme } = useTheme();
+  const [searchType, setSearchType] = useState('query');
+
+  const handleSearchTypeChange = event => {
+    const selectedType = event.target.value;
+    setSearchType(selectedType);
+  };
+
+  return (
+    <div>
+      <label className={styles.searchTypeSelector}>
+        <p
+          className={styles.searchText}
+          //className={`${styles.searchText} ${
+          //theme === 'dark' ? styles.darkTheme : ''
+          //  }`}
+        >
+          Search by:{' '}
```

Comment:
❓

Reply:
The space between the curly braces ({}) is inserted as a literal space character in the rendered output to ensure consistent spacing between elements and text. Otherwise JSX treats everything inside {} as a JavaScript expression.

Reply:
✅